### PR TITLE
feat: Landing Page - Everything you need to build onchain section

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**

Add new "Everything you need to build onchain" section to the landing page. 
- This highlights and contains a link to the relevant documentation for the following components: Frame, Identity, Wallet, Tokens, and Swap. 

**Notes to reviewers**

**How has it been tested?**
Screenshot 2024-06-21 at 3 40 09 PM

Screenshot 2024-06-21 at 3 40 19 PM

Screenshot 2024-06-21 at 4 03 54 PM

### CODE REVIEW

File: site/docs/pages/index.mdx [hunk 1]
```diff
@@ -194,7 +270,7 @@ setFilteredTokens(filteredTokens);
 <hr className="my-8 md:my-20" />
 
 <section>
-  <main className="flex flex-row gap-1">
+  <main className="flex flex-row gap-1 font-normal">
```
Comment (cpcramer) [inline site/docs/pages/index.mdx:273]: Updating the weight of all major titles on the landing page to resemble the newly added `Everything you need to build onchain` section and Figma mockups.

File: site/docs/pages/index.mdx [hunk 2]
```diff
@@ -58,7 +58,7 @@ bun add @coinbase/onchainkit
 :::
 
       </div>
-      <a href="/getting-started" className="font-sans justify-self-end border rounded-xl border-solid border-white px-4 py-3 font-bold">
+      <a href="/getting-started" className="font-sans justify-self-end border rounded-xl border-solid border-white px-3 py-2 font-bold">
```
Comment (kyhyco) [inline site/docs/pages/index.mdx:61]: missing `target="_blank"`

File: site/docs/pages/index.mdx [hunk 3]
```diff
@@ -70,9 +70,39 @@ bun add @coinbase/onchainkit
...
+    ))}
+  </div>
+</section>
 
+<hr className="md:mb-16 md:mt-32" />
+
+{' '}
 <section>
-  <main className="flex flex-col gap-4 md:flex-row">
+  <main className="flex flex-col gap-4 md:flex-row font-normal">
```
Comment (kyhyco) [inline site/docs/pages/index.mdx:105]: nit: add the `font-normal` directly in `h2`

File: site/docs/pages/index.mdx (hunk 1 above)
Comment (kyhyco) [reply site/docs/pages/index.mdx:227]: ditto

Comment (kyhyco) [reply site/docs/pages/index.mdx:227]: and for the once below

**Review by kyhyco — APPROVED** [review]

**Pull request merged**